build.gradle

```diff
@@ -0,0 +1,40 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    
+    dependencies {
+        classpath 'org.akhikhl.gretty:gretty:+'
+    }
+}
+
+plugins {
+    id 'java'
+}
+
+apply plugin: 'war'
+
+apply plugin: 'org.akhikhl.gretty'
+
+group 'HelloServlet'
+
+version '1.0-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile group: 'javax.servlet', name: 'javax.servlet-api', version: '3.1.0'
+    testCompile 'junit:junit:4.12'
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+    compile group: 'javax.servlet', name: 'jstl', version: '1.2'
```

Comment:
Let's define dependencies in one way.
 `group: 'group', name: 'name', version: 'version'`
 `'group:name:version'`
are the same records. I would recommend you to use second one.
Also it is good practice to have compile dependencies at the top and testCompile at the bottom.
And there is still 2 dependencies with junit, remove one of them please